deploy/infrastructure/terraform/.env

```diff
@@ -0,0 +1,35 @@
+# Can be set to your own hook builds
+vOSIE=v0.7.0
```

Comment:
mind using the v0.8.0 version here?

Reply:
Absolutely! I'll upgrade and run through my tests again :+1: